db/schema.yml

```diff
@@ -4541,6 +4541,17 @@ middleware_domains:
 - ems_id
 - created_at
 - updated_at
+middleware_messagings:
```

Comment:
I see others have committed to this file.  I think that's ok. @carbonin can you confirm?